src/main/java/tools/map/making/ConnectionFinder.java

```diff
@@ -364,17 +364,17 @@ public static Shape scale(final Shape currentShape, final int pixels) {
    * Returns a scaled version of the given shape, calculated by the given scale factor.
    * The scaling will be calculated around the centroid of the shape.
    *
-   * @param xScaleFactor
+   * @param sx
    *        how much to scale on the x-axis
-   * @param yScaleFactor
+   * @param sy
    *        how much to scale on the y-axis
    * @return a scaled version of the given shape, calculated around the centroid by the given scale factors.
    */
-  private static Shape scale(final Shape currentPolygon, final double xScaleFactor, final double yScaleFactor) {
+  private static Shape scale(final Shape currentPolygon, final double sx, final double sy) {
```

Comment:
`sx` and `sy` are the parameter names used by `AffineTransform#scale()`.  I just copied those names here.